protected/models/NotNode.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Class OrNode
+ */
+class NotNode extends LogicNode
+{
+    public function renderNode($id)
```

Comment:
Preferably this should be handled elsewhere

Reply:
Have we determined how nodes are going to be rendered on-screen? It would help to have a clear picture before defining the structure.

Reply:
For now at least this fits the pattern of the existing nodes/parameters